src/jdk.management.agent/share/conf/management.properties

```diff
@@ -302,3 +302,5 @@
 #   If the pattern ends with "*", it matches any class with the pattern as a prefix.
 #   If the pattern is equal to the class name, it matches.
 #   Otherwise, the status is UNDECIDED.
+com.sun.management.jmxremote.serial.filter.pattern=java.lang.*;java.math.BigInteger;java.math.BigDecimal;java.util.*;javax.management.openmbean.*;javax.management.ObjectName;java.rmi.MarshalledObject;javax.security.auth.Subject;!*
+
```

Comment:
Should the pattern also include the name of the module that define these type?
Also this set doesn't seem to include javax.management.MBeanInfo and all the types it might contain. This leads me to think that regular MBeans which are not MXBeans or OpenMBeans, like the `DiagnosticCommandMBean` or the `MBeanServerDelegateMBean` might become inaccessible from remote after this change.

Reply:
Thinking that won't matter for packages like java.lang but we could mention somewhere (e.g. release note) that a package can be specified (it's documented in the ObjectInputFilter api docs I think).

Reply:
Yes. It would matter for any package not in the java space that has a pattern allowing recursive match, like e.g. `<package-prefix>.**` ; In that case specifying the module would be important.